### ISSUE TITLE
Original font reader

### ISSUE THREAD

**Issue by Toxaris-NL** [acivux/SvgMath#4]
I have translated the original TTF reader for the font info into C#. Would you be interested in that for incorporation instead of the XML files?

**Comment by acivux** [comment]
I made a commit yesterday that includes the ported [ttf reader](https://github.com/acivux/SvgMath/blob/master/SvgMath/FontLoader/TrueTypeFont.cs). Feel free to look at it, and if you think it is junk, let me know, and I we can get your version in. I have no strong attachment to my version of the reader :)

**Comment by Toxaris-NL** [comment]
I saw that commit, but from what I saw it will still need the XML files. The bit I did no longer needs the font XML files, it can retrieve it from the TTF files, just like the orignal program.

**Comment by acivux** [comment]
No, you have a [choice](https://github.com/acivux/SvgMath/blob/master/SvgMath/Configuration/MathConfig.cs#L34-L37)
